docs/PacketFence_Upgrade_Guide.asciidoc

```diff
@@ -1603,9 +1603,6 @@ The option `NTLM cache background job` and its associated parameters have been d
 
 The `pf-maint.pl` script used to get maintenance patches has been deprecated. You can now get maintenance patches using your package manager, see <<#_apply_maintenance_patches,Apply maintenance patches section>>.
 
-=== WMI scan engine deprecated
```

Comment:
We should keep that in the upgrade guide - because otherwise if someone does upgrade from an old version of PF and doesn't run /usr/local/pf/addons/upgrade/to-11.1-remove-wmi-scan.pl it might break their PacketFence installation.

Reply:
it should be ok now ?